libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -724,33 +720,6 @@ void AP_InertialSensor_Backend::update_gyro(uint8_t instance) /* front end */
         _imu._gyro_filter[instance].set_cutoff_frequency(_gyro_raw_sample_rate(instance), _gyro_filter_cutoff());
         _last_gyro_filter_hz = _gyro_filter_cutoff();
     }
-
-    // possily update the harmonic notch filter parameters
```

Comment:
See my comment about the move of backend to frontend data above

Reply:
same